gbpservice/nfp/configurator/test/lb_rpc_sender_v1.py

```diff
@@ -26,9 +26,9 @@ def __init__(self):
         self.vip = {
                     "status": "ACTIVE",
                     "protocol": "TCP",
-                    "description": {"floating_ip": "192.168.100.149",
-                                    "provider_interface_mac":
-                                        "aa:bb:cc:dd:ee:ff"},
```

Comment:
Can we remove this file and the test directory. Let us discuss on where to keep this. But, please remove from here are it is causing issues in pep8 and flake8 tests.